base_classes/NXlog.nxdl.xml

```diff
@@ -109,6 +109,9 @@
 	  <attribute name="start" type="NX_DATE_TIME">
 			<doc>If missing start is assumed to be the same as for "time".</doc>
 		</attribute>
+		<attribute name="scaling_factor" type="NX_NUMBER" />
```

Comment:
Here is the problem.  Since there is content in the *attribute* element, the tag must be left open on this line.  

Change ` />` to `>`.

If that's the only problem, this correction will fix it.